src/components/MapView/MapView.tsx

```diff
@@ -52,18 +52,88 @@ const FIT_BOUNDS_OPTIONS = {
   padding: 20
 }
 
+type TableRowProps = {
+  name: string
+  value: number
+}
+
+const TableRow = ({ name, value }: TableRowProps) => (
+  <tr>
+    <td>{name}</td>
+    <td style={{ paddingLeft: 15 }}>{value}</td>
+  </tr>
+)
+
+function providerPopup(point: any) {
+  console.log(point)
+  /**
+   * TODO: expose info for all providers with the same address,
+   * not only the first one.
+   */
+  let pointProps = point.features[0].properties
+  return (
+    <div className='popup-container'>
+      <Popup
+        offset={[0, -20]}
+        anchor='bottom'
+        coordinates={point.lngLat} >
+        <table className='popup-table'>
+          <tbody>
+            <TableRow name='Address' value={pointProps.address} />
+            <TableRow name='Specialty' value={pointProps.specialty} />
+            <TableRow name='Lat' value={point.lngLat['lat'].toFixed(4)} />
```

Comment:
Nit: In JS it's better to use dot notation where possible (it's not Python where brackets are mandatory): `point.lngLat.lat.`.

Also, consider pulling the `.toFixed(4)` out into formatters.ts.

Reply:
Done